Element84/filmdrop-ui
# Grid-code aggregation pattern error

**Related Issue(s):**

- NA

**Proposed Changes:**

1. Fixes bug by ignoring grid-code keys that don't match `mapGridCodeFromJson` mapping pattern

### To Test:

> confirm automated tests pass 🍏 

- set config to use API that has non conforming pattern present in aggregations results (see PR owner for value if needed)
- set config value for `SEARCH_MIN_ZOOM_LEVELS` for the collection in question with `"medium": 1`
- load app
- select collection with known non conforming value
- set start date to beginning of collection
- set cloud cover to max so all results returned
- zoom out to low zoom level (not max zoom out, but far enough to see whole world)
- hit search
- confirm results are returned on map for grid code aggregations as expected
- open dev tools and select network tab
- hit search again
- follow last search request URL to view API response in new browser tab
- do a control find on the request URL page
- confirm key of `none` is present in results

**PR Checklist:**
